providers/universal-provider/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@walletconnect/universal-provider",
+  "description": "Universal Provider for WalletConnect Protocol",
+  "version": "2.0.0-rc.1",
+  "author": "WalletConnect, Inc. <walletconnect.com>",
+  "homepage": "https://github.com/walletconnect/walletconnect-monorepo/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/walletconnect/walletconnect-monorepo",
+    "directory": "packages/universal-provider"
+  },
+  "license": "Apache-2.0",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.es.js",
+  "unpkg": "dist/index.umd.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "wallet",
+    "walletconnect"
+  ],
+  "scripts": {
+    "clean": "rm -rf dist",
+    "build:pre": "npm run clean",
+    "build:types": "tsc",
+    "build:source": "rollup --config rollup.config.js",
+    "build": "npm run build:pre; npm run build:source; npm run build:types",
+    "test:pre": "rm -rf ./test/test.db",
+    "test:run": "vitest run --dir test",
+    "test": "npm run test:pre; npm run test:run",
+    "lint": "eslint -c '../../.eslintrc' --fix './src/**/*.ts'",
+    "prettier": "prettier --check '{src,test}/**/*.{js,ts,jsx,tsx}'"
+  },
+  "dependencies": {
```

Comment:
I think some of these dependencies are only needed for testing purposes, correct? If that is the case, could you move relevant ones to `devDependencies` so they don't reflect as deps in npm registry. (re install them to update pakage lock file as well)

Reply:
> Looking goood fire Will need to wait for tests to get implemented for final approval. One question, what's the idea behind `subProviders` folder? Is it to simplify implementation even further so that we can create those small chain configs ourselves?

Think of `subProviders` as controllers. Each namespace will have its own implementation where logic can't be shared between them.